### ISSUE TITLE
Adding Data for MARIA Antigen Presentation Prediction Support

### ISSUE THREAD

**Issue by alat-rights** [deepchem/deepchem#2229]
Hi! 

I've been working on integrating this work ( https://www.nature.com/articles/s41587-019-0280-2 ) in Deepchem. 

Our first step is to add the relevant datasets to Moleculenet. 

The datasets used in the paper were:
- Patient MS HLA-DR Ligands
- Public HLA-2 peptide binding data
- Sequencing for HLA-2 allel and gene expression levels. 

All of the data is publicly available, and access instructions are given in the linked article. 

I am currently working on adding this dataset from PRIDE archive https://www.ebi.ac.uk/pride/archive/projects/PXD004746 to Moleculnet. I could use a bit of help here, since I'm not really sure how to interpret the RAW files. Parts of it are human readable but there are large parts that look encoded.

**Comment by alat-rights** [comment]
Maybe the data here https://www.ebi.ac.uk/pride/archive/projects/PXD004746 maybe should be processed through PEAKS or some other external tool before being added to MolNet?

**Comment by alat-rights** [comment]
I'm working on getting the IEDB bonding data into our S3 first just so that we have something to work with! 

The search restriction is applied here: https://www.iedb.org/result_v3.php?cookie_id=766381

There is data on 23077 T-Cell assays and 253500 MHC Ligand Assays for HLA-DR protein complex. I'm not sure how to further restrict / which assays to use. I could use a little bit of advice if possible!

**Comment by rbharath** [comment]
Hmm, so my initial instinct here is that if there isn't a clear restriction we should just make the full dataset available. If we document the source clearly with references, downstream users can filter based on their needs

**Comment by alat-rights** [comment]
Sounds good! So the whole thing goes into MolNet?

**Comment by rbharath** [comment]
Yes I think so! As a quick check, how big is the full dataset? So long as it's under say 10 GB we don't have to think too much about it I think

**Comment by alat-rights** [comment]
80~A few hundred megs zipped. Any chance I could fetch the file on the AWS instance so I won't have to download+reupload?

**Comment by rbharath** [comment]
So all our storage is on S3 (https://deepchem.readthedocs.io/en/latest/infra.html#s3), so unfortunately no clean way to do this I think? There might be something clever to do with the AWS command line tools but I've usually just downloaded and uploaded.

**Comment by alat-rights** [comment]
Ok! I just have to wait for when I have a lot of time to babysit a download. Sorry for dropping the ball on this a little, it's on the todo
